src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.SupportedTargetPlatforms.props

```diff
@@ -0,0 +1,26 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.SupportedTargetPlatforms.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+
+<!-- This file contains a list of the target platforms that are supported by this SDK for .NET. -->
```

Comment:
This file should be Windows-specific.  The iOS and Android supported versions would be included in props/targets files in the corresponding workloads.  (Once Windows is a separate workload, this file would move into that workload also.)